githubutils/repo.go

```diff
@@ -33,6 +34,8 @@ const (
 
 	CONTENT_TYPE_FILE      = "file"
 	CONTENT_TYPE_DIRECTORY = "dir"
+
+	MAX_GITHUB_RELEASES_PER_PAGE = 100
```

Comment:
Why do we change the max release per page from 10,000,000 (https://github.com/solo-io/go-utils/pull/449/files#diff-18395ae6cd3dd026be8823a5785e272b7b12f3214e02169d1039ff2f32a48767L164) to 100?

Reply:
https://docs.github.com/en/rest/overview/resources-in-the-rest-api#pagination
> For some resources, you can also set a custom page size up to 100 with the per_page parameter